website/addons/wiki/templates/edit.mako

```diff
@@ -4,6 +4,13 @@
 ## Use full page width
 <%def name="container_class()">container-xxl</%def>
 
+% if user['can_comment'] or node['has_comments']:
+    % if page:
+        <%include file="include/comment_pane_template.mako"/>
+        <%include file="include/comment_template.mako"/>
```

Comment:
`comment_pane_template.mako` already includes `comment_template.mako`, so I believe this `include` is redundant.